src/containers/activity/activity-area.tsx

```diff
@@ -0,0 +1,126 @@
+import Anchor from '@ui/anchor';
+import { getTimeAgo } from '@utils/getTimeAgo';
+import clsx from 'clsx';
+import Image from 'next/image';
+import useSWR from 'swr';
+import { itemService } from '../../features/leda-nft/services/item.service';
+
+type Props = {
+  space?: number;
+  className?: string;
+};
+
+const fetchHistories = () => itemService.findAllHistory().then((data) => data.reverse());
+
+export const ActivityArea = ({ space = 1, className }: Props) => {
+  const { data: history } = useSWR('fetchAllHistory', fetchHistories);
+  // TODO: Enable filters if needed
```

Comment:
Not sure if this filters should go here, but I'd suggest to drop this commented code.